matchms/filtering/require_precursor_below_mz.py

```diff
@@ -0,0 +1,31 @@
+from ..typing import SpectrumType
+
+
+def require_precursor_below_mz(spectrum_in: SpectrumType, max_mz: float = 1000) -> SpectrumType:
+
+    """Returns None if the precursor_mz of a spectrum is above
+       max_mz.
+
+    Parameters
+    ----------
+    spectrum
```

Comment:
Match docstring with function spec
```suggestion
    spectrum_in
```